docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+[id="installing-intellij-idea-plugin_{context}"]
+= Installing the {ProductShortName} plugin for IntelliJ IDEA
+
+You can install the {ProductShortName} plugin in both IntelliJ IDEA Ultimate and the IDEA Community Edition.
```

Comment:
I don't think you need "the" before the second IDEA. Or else add it to the first IDEA. Having only one definite article for two parallel entities looks odd.